open_alchemy/__init__.py

```diff
@@ -270,24 +280,35 @@ def build_json(
     with open(spec_filename) as spec_file:
         spec = json.load(spec_file)
 
-    return _build_module.execute(spec=spec, name=package_name, path=dist_path)
+    return _build_module.execute(
+        spec=spec, name=package_name, path=dist_path, format_=format_
+    )
 
 
 def build_yaml(
     spec_filename: str,
     package_name: str,
     dist_path: str,
+    format_: PackageFormat = PackageFormat.NONE,
 ) -> None:
     """
     Create an OpenAlchemy distribution package with the SQLAlchemy models.
 
     The package can be uploaded to, for example, PyPI or a private repository for
     distribution.
 
+    The formats can be combined with the bitwise operator "or" (``|``), for
+    instance, building both sdist and wheel packages can be specified like that:
+
+    .. code-block:: python
+
+        format_=PackageFormat.SDIST|PackageFormat.WHEEL
+
     Args:
         spec_filename: filename of an OpenAPI spec in YAML format
         package_name: The name of the package.
         dist_path: The directory to output the package to.
+        format_: The format(s) of the archive to build if any
```

Comment:
Same as above, please note that the parameter is optional.

Reply:
Addressed ✅